src/utils.js

```diff
@@ -57,7 +57,7 @@ const utils = {
         if (data.event.release) {
             parts.push(`<span title="release">${data.event.release}</span>`, '|');
         }
-        parts.push(`<a href="${data.url}">${data.event.title || data.message}</a>`);
+        parts.push(`${data.event.title || data.message} (<a href="${data.url}">event</a>)`);
```

Comment:
Accessibility note:
For users without sight, every link should have a distinct label.

Reply:
I'm not sure this is permitted by Matrix clients [based on spec](https://spec.matrix.org/unstable/client-server-api/#mroommessage-msgtypes) :thinking:

Reply:
Not by using `<label>` but by using
- `<a>event A</a>`
- `<a>event B</a>`
- `<a>event C</a>`
- `<a>event D</a>`

instead of
- `A happened: <a>event</a>`
- `B happened: <a>event</a>`
- `C happened: <a>event</a>`
- `D happened: <a>event</a>`

In "link"-mode screen readers will only read the content of a link and that would be "event" every time.

I'm not urging you to change this now. This looks like an internal event log to me.
It's something to keep in mind and adhere to whenever possible (like `alt` on an `<img>`).
If someone with a screen reader or a (government) agency ever has to work with our software, they'll have a better time.